web/src/main/java/org/cbioportal/web/DataAccessTokenController.java

```diff
@@ -88,7 +79,13 @@ private void initializeUsersWhoCannotUseTokenSet() {
                                     @RequestParam(required = false) Boolean allowRevocationOfOtherTokens) throws HttpClientErrorException {
```

Comment:
Can we use `-` separated paths in all endpoints to make it consistent with all other APIs? And also can we use plural for all resource names in the path, like `data-access-tokens`.

Reply:
@ersinciftci done